unit_tests/multiphase/test_vof_plic.cpp

```diff
@@ -136,6 +234,153 @@ amrex::Real normal_vector_test_impl(amr_wind::Field& vof, const int dir)
     return error_total;
 }
 
+amrex::Real normal_vector_neumann_test_impl(
+    amr_wind::Field& vof, amr_wind::IntField& iblk_fld)
+{
+    amrex::Real error_total = 0.0;
+
+    for (int lev = 0; lev < vof.repo().num_active_levels(); ++lev) {
+
+        error_total += amrex::ReduceSum(
+            vof(lev), iblk_fld(lev), 0,
+            [=] AMREX_GPU_HOST_DEVICE(
+                amrex::Box const& bx,
+                amrex::Array4<amrex::Real const> const& vof_arr,
+                amrex::Array4<int const> const& iblank) -> amrex::Real {
+                amrex::Real error = 0.0;
+
+                amrex::Loop(bx, [=, &error](int i, int j, int k) noexcept {
+                    int ibdy = 0;
+                    int jbdy = 0;
+                    int kbdy = 0;
+                    if (iblank(i, j, k) != iblank(i - 1, j, k)) {
```

Comment:
I think we can consolidate these:
const int ibdy = (iblank(i, j, k) != iblank(i - 1, j, k)) ? -1 : 0;